lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -123,9 +118,9 @@ static SPIRVMemoryModelKind getMemoryModel(Module &M) {
   return SPIRVMemoryModelKind::MemoryModelMax;
 }
 
-LLVMToSPIRV::LLVMToSPIRV(SPIRVModule *SMod)
+LLVMToSPIRV::LLVMToSPIRV(SPIRVModule *SMod, bool SPIRVAllowUnknownIntrinsics)
```

Comment:
I wonder why this change is needed? Please note that `TranslatorOpts` instance is available within `SPIRVModule` and you can query desired property from the module instead.

For example:

https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/2846475f19c56ae034d6e418fac48b117ab68bc2/lib/SPIRV/libSPIRV/SPIRVModule.h#L467-L469

Reply:
Fixed